analytics_data_api/v0/models.py

```diff
@@ -148,8 +148,23 @@ class Meta(BaseCourseEnrollment.Meta):
         unique_together = [('course_id', 'date', 'country_code')]
 
 
+class CourseGradeBreakdown(models.Model):
+    """ The number of students who have a given letter grade for a given course and date """
+
+    class Meta(object):
+        db_table = 'grade_breakdown'
+
+    course_id = models.CharField(db_index=True, max_length=255)
+    date = models.DateField(null=False)
+    letter_grade = models.CharField(max_length=64, null=True)
+    num_students = models.IntegerField(null=False)
+    percent_students = models.FloatField()
+    is_passing = models.BooleanField(null=False)
+    created = models.DateTimeField(auto_now_add=True)
+
+
 class GradeDistribution(models.Model):
-    """ Each row stores the count of a particular grade on a module for a given course. """
+    """ Each row stores the count of a particular score on a module for a given course. """
```

Comment:
@bradenmacdonald If this model represents a score distribution, maybe the terminology should be changed in the rest of the class as well? Not sure if you would want to do that as part of this PR (it would probably exceed the scope of the PR a bit), but I wanted to point it out just in case.